test/e2e/scheduling/limit_range.go

```diff
@@ -183,144 +207,18 @@ var _ = SIGDescribe("LimitRange", func() {
+		pod = f.NewTestPod(podName+"2", getResourceList("600m", "600Mi", "600Mi"), v1.ResourceList{})
 		pod, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(pod)
 		Expect(err).NotTo(HaveOccurred())
-
 		defer func() {
 			By("Removing pod")
-			err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Delete(pod.Name, nil)
+			err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Delete(podName+"2", nil)
 			Expect(err).NotTo(HaveOccurred())
 		}()
-
-		By("Ensuring Pod has merged resource requirements applied from LimitRange")
-		pod, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Get(pod.Name, metav1.GetOptions{})
-		Expect(err).NotTo(HaveOccurred())
-		// If you specify a Request, and no Limit, the Limit will be set to default limit
-		expected = v1.ResourceRequirements{Requests: getEphemeralStorageResourceList("150Mi"), Limits: defaultLimit}
-		for i := range pod.Spec.Containers {
-			err = equalResourceRequirement(expected, pod.Spec.Containers[i].Resources)
-			if err != nil {
-				// Print the pod to help in debugging.
-				framework.Logf("Pod %+v does not have the expected requirements", pod)
-				Expect(err).NotTo(HaveOccurred())
-			}
-		}
-
-		By("Creating a Pod with limit")
-		pod = f.NewTestPod("pod-partial-resources", v1.ResourceList{}, getEphemeralStorageResourceList("300m"))
-		pod, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(pod)
-		Expect(err).NotTo(HaveOccurred())
-
-		defer func() {
-			By("Removing pod")
-			err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Delete(pod.Name, nil)
-			Expect(err).NotTo(HaveOccurred())
-		}()
-
-		By("Ensuring Pod has merged resource requirements applied from LimitRange")
-		pod, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Get(pod.Name, metav1.GetOptions{})
-		Expect(err).NotTo(HaveOccurred())
-		// If you specify a Limit, and no Request, the Limit will default to the Request
-		// This means that the LimitRange.DefaultRequest will ONLY take affect if a container.resources.limit is not supplied
-		expected = v1.ResourceRequirements{Requests: getEphemeralStorageResourceList("300Mi"), Limits: getEphemeralStorageResourceList("300Mi")}
-		for i := range pod.Spec.Containers {
-			err = equalResourceRequirement(expected, pod.Spec.Containers[i].Resources)
-			if err != nil {
-				// Print the pod to help in debugging.
-				framework.Logf("Pod %+v does not have the expected requirements", pod)
-				Expect(err).NotTo(HaveOccurred())
-			}
-		}
-
-		By("Failing to create a Pod with less than min resources")
-		pod = f.NewTestPod(podName, getEphemeralStorageResourceList("50Mi"), v1.ResourceList{})
-		pod, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(pod)
-		Expect(err).To(HaveOccurred())
-
-		By("Failing to create a Pod with more than max resources")
-		pod = f.NewTestPod(podName, getEphemeralStorageResourceList("600Mi"), v1.ResourceList{})
-		pod, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(pod)
-		Expect(err).To(HaveOccurred())
```

Comment:
You are removing this test for ephemeral storage?

Reply:
no. i do not remove them. just merge local ephemeral storage with CPU and memory test cases Since local ephemeral storage feature is enabled by default